apps/kyc-server/docs/CORS.md

```
@@ -1,76 +1,76 @@
+### 1. Environment Variables
+
+The simplest way to configure CORS is through the `ALLOWED_ORIGINS` environment variable in your `.env` file:
+
+```
+# Comma-separated list of allowed origins (no spaces)
+ALLOWED_ORIGINS=https://app.example.com,https://admin.example.com
+```
+
+In development mode, you can use a wildcard to allow all origins:
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)